From b910f4b38800e832766cbb1f15dec57a820c3776 Mon Sep 17 00:00:00 2001
From: Han Wang <416810799@qq.com>
Date: Tue, 25 Sep 2018 14:36:18 +0200
Subject: [PATCH 37/45] ConnectivityManager: Change default private DNS mode

* I really don't know if even a single person uses this, and this causes
  a lot of warnings related to netd in logcat.

Change-Id: I3ea689bbedd06650158f30d21c2b0b4550fd4ac0
---
 core/java/android/net/ConnectivityManager.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/net/ConnectivityManager.java b/core/java/android/net/ConnectivityManager.java
index c5cb1f5b7cf..8597b3cc136 100644
--- a/core/java/android/net/ConnectivityManager.java
+++ b/core/java/android/net/ConnectivityManager.java
@@ -704,7 +704,7 @@ public class ConnectivityManager {
      *
      * @hide
      */
-    public static final String PRIVATE_DNS_DEFAULT_MODE_FALLBACK = PRIVATE_DNS_MODE_OPPORTUNISTIC;
+    public static final String PRIVATE_DNS_DEFAULT_MODE_FALLBACK = PRIVATE_DNS_MODE_OFF;
 
     private final IConnectivityManager mService;
     /**
-- 
2.17.1

